ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -31,4 +31,39 @@
 
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("testing class azure_connection", "azure_connection")
+{
+
```

Comment:
Pls wrap the unit tests for read_objects in a SECTION

Reply:
resolved https://github.com/wiredtiger/wiredtiger/pull/8738/commits/62f7e4766feef2010b903baa8e78a7234e1274da